admin/src/components/ConfigsTable/index.tsx

```diff
@@ -52,13 +52,16 @@ export default function ConfigsTable() {
         const newData = await get<any, ConfigResponse>(`/${pluginId}/config`);
         setData(newData.data);
         setIsLoading && setIsLoading(false);
-      }
-      catch (err) {
+      } catch (err) {
         console.error(err);
       }
     }
   }
 
+  const toggleConfirmDialog = () => {
+    setIsConformDialogOpen((prev) => !prev);
+  };
+
   const COL_COUNT = 4;

```

Comment:
keep things like this outside of the comp. since they are not effected by any state or data between re-renders inside in the component